lib/SPIRVProducerPass.cpp

```diff
@@ -399,7 +399,9 @@ struct SPIRVProducerPassImpl {
   SmallPtrSet<Value *, 16> &getGlobalConstArgSet() {
     return GlobalConstArgumentSet;
   }
-  StrideTypeList &getTypesNeedingArrayStride() { return TypesNeedingArrayStride; }
```

Comment:
I think this is comming from clang-format, but as you do not modify it, you should not have to commit that. Please try to revert that.